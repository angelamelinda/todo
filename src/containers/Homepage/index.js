import React, { Component } from 'react';

import ListTask from '../../components/ListTask';
import AddTask from '../../components/AddTask';

class Homepage extends Component {
    render() {
        return (
            <React.Fragment>
                <AddTask />
                <ListTask />
            </React.Fragment>      
        )
    }
}

export default Homepage;