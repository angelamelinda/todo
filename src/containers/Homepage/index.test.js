import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

import Homepage from './';

Enzyme.configure({ adapter: new Adapter()});

describe('Homepage', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(
      <Homepage />
    )
    expect(toJson(wrapper), {mode:'shallow'}).toMatchSnapshot();
  });
})
