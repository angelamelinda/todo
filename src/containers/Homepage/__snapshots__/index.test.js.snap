// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage renders without crashing 1`] = `
<Fragment>
  <Connect(AddTask)
    dataEdit={Object {}}
  />
  <Connect(ListTask)
    edit={[Function]}
  />
</Fragment>
`;
